packages/web/src/components/buy-audio-modal/components/AmountInputPage.tsx

```diff
@@ -48,13 +51,11 @@ export const AmountInputPage = () => {
       />
       <PurchaseQuote />
       <div className={styles.buyButtonContainer}>
-        <CoinbaseBuyAudioButton
-          amount={
-            purchaseInfo?.isError === false
-              ? purchaseInfo.estimatedSOL.uiAmount
-              : undefined
-          }
-        />
+        {provider === OnRampProvider.COINBASE ? (
+          <CoinbaseBuyAudioButton />
+        ) : (
+          <StripeBuyAudioButton />
```

Comment:
provider is optional as a prop, so if provider is not passed in then we use Stripe by default. just double-checking that it's what we want

Reply:
good point, I think i'll let it be selected in this component rather than passed via a prop (but yeah, I think defaulting to stripe makes sense